modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -319,6 +321,13 @@ def lob
       }[benefit_type]
     end
 
+    def assign_metadata
+      # retain original incoming non-pii form_data in metadata since this model's form_data is eventually removed
+      self.metadata = { form_data: { consumer_benefit_type: benefit_type } }
```

Comment:
I could be wrong, but I thought we were going to call it `benefit_type` to match the field in the schema, since the fact that it's under `form_data` already points to the fact that it's from the consumer.

Reply:
Yeah, sorry for the confusion. form_data keys should try to match their schema counterparts relatively closely - if we were to also add a new root-level key/group specifically for tracking the received/sent business lines, then a custom key without influence form the schema name would be the way to go there.

In this case I think just having the form_data.benefit_type and a separate central_mail_business_line is fine, but if it feels like copying the value into a new root key would be better for querying down the line, I'm not opposed.